server-phoenix/lib/helios_web/controllers/web_hooks/slack_controller.ex

```diff
@@ -1,12 +1,134 @@
 defmodule HeliosWeb.WebHooks.SlackController do
   use HeliosWeb, :controller
-  alias Helios.{Repo, Event}
+  require Logger
+  alias Helios.{Repo, Event, SlackChannelNames}
+  defp slack_bearer_token, do: System.get_env("SLACK_BEARER_TOKEN")
```

Comment:
hmmmm, I think the pattern for keeping tokens is in the `config` folder over here https://github.com/mojotech/helios2/blob/master/server-phoenix/config/config.exs

Reply:
fixed this up in the next commit